### ISSUE TITLE
Outdoor Environment Camera Pose

### ISSUE THREAD

**Issue by tb2-sy** [marek-simonik/record3d#64]
Is it reasonable to use record3d to obtain the camera pose in an outdoor environment? Because there are no cases of outdoor environments in many examples.

**Comment by marek-simonik** [comment]
The LiDAR poses provided by Record3D are (currently) estimated by ARKit, which seems to work reasonably well in both indoor and outdoor environment. There is no particular reason why I have not yet shown a sample outdoor LiDAR video.

**Comment by TimYao18** [comment]
Are the poses converted into OpenGL coordinate system from ARKit `arframe.camera.transform` when recording videos?

**Comment by marek-simonik** [comment]
> Are the poses converted into OpenGL coordinate system from ARKit `arframe.camera.transform` when recording videos?

Yes, they are; the poses reported by Record3D use the OpenGL coordinate system convention.

**Comment by marek-simonik** [comment]
> Thanks for the great app, with ARKit do you mean the apple's ARKit https://developer.apple.com/documentation/arkit/arframe ?

Thank you for using Record3D. Yes, I do — exactly as you wrote.
